### PR TITLE
Add Matomo template

### PR DESCRIPTION
**Pull request by trispera**
Creation of a Helm Chart regarding previous Matomo template from Rahti 1

### CODE REVIEW

File: charts/matomo/values.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  service:
+    type: ClusterIP
+  pvc:
+    storageSize: 5Gi
+    storageClassName: standard-csi
+  secret:
+    databaseName: matomodb
+    databaseUser: matomouser
+    databasePassword: letmein
+    rootPassword: letmein
```
Comment (lvarin) [inline charts/matomo/values.yaml:13]: Could these password be random?

File: charts/matomo/values.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  name: matomo
+  service:
+    type: ClusterIP
+  route:
+    tls:
+      insecureEdgeTerminationPolicy: Redirect
+      termination: edge
+  secret:
+    matomoUser: matomouser
+    matomoPassword: letmein
```
Comment (lvarin) [inline charts/matomo/values.yaml:37]: Again, could we have random password as well?

File: charts/matomo/templates/deploymentconfig.yaml [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        persistentVolumeClaim:
+          claimName: {{ .Values.mariadb.name }}
+  triggers:
+  - imageChangeParams:
+      automatic: true
+      containerNames:
+      - mariadb
+      from:
+        kind: ImageStreamTag
+        name: mariadb:10.3-el8
```
Comment (lvarin) [inline charts/matomo/templates/deploymentconfig.yaml:80]: Maybe it could be a parameter, instead of hardwired version.

**Review by lvarin — CHANGES_REQUESTED** [review]
LGTM, but might be nice to improve the few suggestions

File: charts/matomo/values.yaml (hunk 2 above)
Comment (trispera) [reply charts/matomo/values.yaml]: Fixed. Password are defined randomly in the `_helpers.tpl` file

File: charts/matomo/values.yaml (hunk 1 above)
Comment (trispera) [reply charts/matomo/values.yaml]: Fixed. Password are defined randomly in the `_helpers.tpl` file

File: charts/matomo/templates/deploymentconfig.yaml (hunk 3 above)
Comment (trispera) [reply charts/matomo/templates/deploymentconfig.yaml]: Now a parameter in `values.yaml`

**Review by lvarin — APPROVED** [review]

**Pull request merged**